CHANGELOG.md

```diff
@@ -1,12 +1,30 @@
 # Changelog
 
-## unreleased
+## 3.4.0-rc.0 / 2025-04-29
 
 * [CHANGE] Make setting out-of-order native histograms feature (`--enable-feature=ooo-native-histograms`) a no-op. Out-of-order native histograms are now always enabled when `out_of_order_time_window` is greater than zero and `--enable-feature=native-histograms` is set. #16207
 * [FEATURE] OTLP translate: Add feature flag for optionally translating OTel explicit bucket histograms into native histograms with custom buckets. #15850
 * [FEATURE] OTLP translate: Add option to receive OTLP metrics without translating names or attributes. #16441
+* [FEATURE] Promltest: support to test expectation annotations. #15995
+* [FEATURE] PromQL: allow arithmetic operations in durations in PromQL parser. #16249
+* [FEATURE] OTLP: Add primitive support for ingesting OTLP delta metrics as-is. #16360
+* [ENHANCEMENT] promql: histogram_fraction for bucket histograms. #16095
 * [ENHANCEMENT] TSDB: add `prometheus_tsdb_wal_replay_unknown_refs_total` and `prometheus_tsdb_wbl_replay_unknown_refs_total` metrics to track unknown series references during WAL/WBL replay. #16166
-* [BUGFIX] TSDB: fix unknown series errors and possible lost data during WAL replay when series are removed from the head due to inactivity and reappear before the next WAL checkpoint. #16060
+* [ENHANCEMENT] scrape: Add config option for escaping scheme request. #16066
+* [ENHANCEMENT] nhcb: Add global config option for convert_classic_histograms_to_nhcb. #16226
+* [ENHANCEMENT] ruler notifier: make batch size configurable (`--alertmanager.notification-batch-size`). #16254
+* [PERF] Remote-write: re-use memory to read WAL data. #16197
```

Comment:
this was cherry-picked and released in 3.3.0 as a bug fix.

Reply:
```suggestion
```